src/Microsoft.AspNet.OData.Shared/IEntityObjectInstanceAnnotations.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Copyright (c) Microsoft Corporation.  All rights reserved.
+// Licensed under the MIT License.  See License.txt in the project root for license information.
+
+using Microsoft.AspNet.OData.Builder;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNet.OData
+{
+    internal static class TransientAnnotations
```

Comment:
>TransientAnnotations [](start = 26, length = 20)

the file name is "IEntityObjectInstanceAnnotation.cs",  but i can't find anything related to this interface in this file?